docs/S3ToRedshift-action.md

```diff
@@ -0,0 +1,68 @@
+# S3ToRedshift Action
+
+
+Description
+-----------
+S3ToRedshift Action that will load the data from AWS S3 bucket into the AWS Redshift table.
+
+Properties
+----------
+
+**accessKey:** Access key for AWS S3 to connect to. (Macro-enabled)
```

Comment:
Added this clause in conditions section in documentation.
Please let me know, if  need to be updated in description for properties also ?